server/rest/controller/v4/schema_controller.go

```diff
@@ -36,15 +36,30 @@ type SchemaService struct {
 }
 
 func (s *SchemaService) URLPatterns() []rest.Route {
+	if !core.ServerInfo.Config.SchemaDisable {
+		return []rest.Route{
+			{Method: rest.HTTPMethodGet, Path: "/v4/:project/registry/microservices/:serviceId/schemas/:schemaId", Func: s.GetSchemas},
```

Comment:
in fact you only need to disable ModifySchema，in another hand this is not clean code.

plz only use core.ServerInfo.Config.SchemaDisable in ModifySchema function